packages/core/src/hooks/usePlate/usePlateEffects.ts

```diff
@@ -49,12 +49,12 @@ export const usePlateEffects = <T extends SPEditor = SPEditor>({
 
   useEffect(() => {
     setInitialState({
-      enabled: true,
+      enabled,
       plugins: [],
       pluginKeys: [],
       value: [],
     });
-  }, [id, setInitialState]);
+  }, [enabled, id, setInitialState]);
```

Comment:
The state will reset each time `enabled` updates, which is not wanted

Reply:
ah, thanks for clarifying.